### DISCUSSION TITLE
Demo Videos

### DISCUSSION THREAD

**Discussion by dbaslan** [SWE574-Fall2023-Group1/SWE574-Fall2023-G1#319] [Ideas]
As you know we need a video or two to demonstrate our app. I am thinking of recording two straightforward demos where one of us uses the app and briefly goes over each feature. I can upload them to YouTube once we're done. Any other ideas or suggestions? @SWE574-Fall2023-Group1/all  See #315

**Comment by sadikkuzu** [comment]
@aykutkantas hi,
Can you record web demo video?

**Reply by aykutkantas** [reply]
Sure @sadikkuzu

**Reply by sadikkuzu** [reply]
Thanks @aykutkantas for your volunteering. I assigned you to #315 task.
Thanks again in advance.
